client/app/scripts/components/node-details/node-details-health-item.js

```diff
@@ -10,8 +10,8 @@ class NodeDetailsHealthItem extends React.Component {
       <div className="node-details-health-item">
       <div className="node-details-health-item-value">{formatMetric(this.props.value, this.props)}</div>
         <div className="node-details-health-item-sparkline">
-          <Sparkline data={this.props.samples} max={this.props.max}
-            first={this.props.first} last={this.props.last} />
+          <Sparkline data={this.props.wire_metrics.samples} max={this.props.wire_metrics.max}
+            first={this.props.wire_metrics.first} last={this.props.wire_metrics.last} />
```

Comment:
That sounds fine. Otherwise, at least for metrics, we just embed it so we can add a few fields. We could just copy all the existing fields over when instantiating the Wire Metrics.